src/admin/settings/settings.html

```diff
@@ -0,0 +1,126 @@
+                <input type="button" ng-click="exportConfig()" value="{{'admin.settings.importexport.exportButton' | translate}}" class="btn btn-default"/>
+            </fieldset>
+        </form>
+    </div>
+</div>
+<div class="clearfix"></div>
+<div class="form-group">
+    <div class="settings-box-content" ng-show="platformSettings.settingsList.length > 0">
+        <legend class="frame-title settings-box-header">{{'admin.settings.platform' | translate}}</legend>
+        <div class="settings-form" ng-repeat="pSettings in platformSettings.settingsList">
+            <legend>{{'admin.settings.section.'+ pSettings.section | translate}}</legend>
+            <form id="platformSettingsForm" class="inside form-horizontal">
+                <fieldset>
+                    <div ng-repeat="option in pSettings.settings" class="form-group">
+                        <label class="desc-label control-label" for="{{option.key}}">
+                            {{'admin.settings.' + option.key | translate}}
+                            <span class="icon-documentation" title="{{'admin.settings.tooltip.' + option.key | translate}}" />
+                        </label>
+                        <div class="settings-input">
+                            <input ng-if="isTextOrNumber(option)" class="form-control input-large" ng-disabled="platformSettings.readOnly" type="text" id="{{option.key}}" name="{{option.key | translate}}" ng-model="option.value" />
+                            <input ng-if="isBool(option)" class="input-checkbox" ng-disabled="platformSettings.readOnly" type="checkbox" id="{{option.key}}" name="{{option.key}}" ng-model="option.value" />
+                            <div ng-if="isCombobox(option)" ng-init="addValue(option.key, option.value)">
+                                <select class="form-control input-large" ng-model="option.value" ng-disabled="platformSettings.readOnly"
+                                    ng-options="value for value in getComboboxValues(option.key)">
+                                    <option ng-if="!option.value" value="">{{'mds.form.label.select' | translate}}</option>
+                                </select>
+                                <input class="input-medium form-control" ng-disabled="platformSettings.readOnly" type="text" ng-model="userValidator" />
+                                <a class="btn btn-success" ng-disabled="!userValidator" ng-click="addValue(option.key, userValidator)">
+                                    <span class="fa fa-fw fa-plus"></span>Add option</a>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <div class="btn-save">
+                            <input ng-if="!platformSettings.readOnly" type="submit" ng-click="saveSettings(pSettings)" value="{{'admin.save' | translate}}" class="btn btn-primary" />
```

Comment:
What is up with the div around the input button? Please remove the div and just have the button, its more semantic.